web/pdf_viewer.css

```diff
@@ -18,7 +18,6 @@
 @import url(annotation_editor_layer_builder.css);
 
 :root {
-  --viewer-container-height: 0;
```

Comment:
This is being used by e.g. the secondaryToolbar as well, so this really can't be removed.